articles/quickstarts/qrng.md

```diff
@@ -48,16 +63,33 @@ We can use this representation to visualize what the code is doing:
 
 <img src="~/media/qrng-H.png" width="450" alt="Preparing a qubit in superposition">
 
-
 * Then we measure the qubit and save the output:
 
 <img src="~/media/qrng-meas.png" width="450" alt="Measuring a qubit and saving the output">
 
 Since the outcome of the measurement is completely random, we have obtained a random bit. We can call this operation several times to create integers. For example, if we call the operation three times to obtain three random bits, we can build random 3-bit numbers (that is, a random number between 0 and 7).
 
-## Creating a complete random number generator using a host program
 
-Now that we have a Q# operation that generates random bits, we can use it to build a complete quantum random number generator with a host program.
+## Creating a complete random number generator
```

Comment:
Actually, where I thought I had seem the section on the actual entry point at some point, but I no longer see it in this diff?

Reply:
You meant `@EntryPoint()`?

Reply:
This won't work as is.

Reply:
The API doc for the `EntryPoint` attribute should be up shortly with the uid `microsoft.quantum.core.entrypoint` (see https://github.com/MicrosoftDocs/quantum-docs-pr/blob/build/0.11.2004.2825/api/qsharp/microsoft.quantum.core.entrypoint.yml).

Reply:
I was looking for the code for the entry point (the entry point implementation). @cgranade I didn't see anywhere where it would be pulled in from the Samples repo, but only saw the code section in this doc that only contains SampleQuantumRandomNumberGenerator. I hence edited that to be correct, and also added a code section for the actual entry point. If it is indeed pulling in the code from Samples now (yay), then should all code sections be removed? Chris, could you take a look at whether the current state makes sense or edit as needed?

Reply:
Can you do the same for the Grover's search? We don't know if the Python compiling will still work with the Entry Point code added.